patches/minecraft/net/minecraft/server/MinecraftServer.java.patch

```diff
@@ -172,6 +172,15 @@
  
        try {
           OptionSet optionset = optionparser.parse(p_main_0_);
+@@ -888,7 +913,7 @@
+ 
+          Path path = Paths.get("server.properties");
+          ServerPropertiesProvider serverpropertiesprovider = new ServerPropertiesProvider(path);
+-         serverpropertiesprovider.func_219035_b();
++         if (optionset.has(optionspec1) || !Files.exists(path)) serverpropertiesprovider.func_219035_b();
```

Comment:
Typically we just inject the IF above the line so the patch file doesn't have a removal line.

Reply:
Noted, thanks